docs/temp-v5issue.html

```diff
@@ -593,7 +593,7 @@ <h2>Reported issue list</h2>
 	<li><strong>Item 86</strong> On iPhone device - Carousel display issue. The navigation bar (arrows) goes over the slide title</li>
 	<li><strong>Item 87</strong> On iPhone device - Top search bar overflow to the right and the search button are render bellow it.</li>
 	<li><del><strong>Fixed Item 88</strong> Multimedia player - Small device and desktop - Review how overlay (with the pause button) are still rendered in the middle and centered.</del></li>
-	<li><del><strong>Fixed Item 89</strong> Form validation - Error label that are appearing when validating checkboxes overlay the legend text. <a href="../demos/formvalid/formvalid-en.html">Form validation</a>. How to reproduce: 1. Go in the form validation example page. 2. Click on submit to trigger the form validation plugin. 3. See the error label are over the legend text. - There probably a fix for it in the PR #8397</del></li>
+	<li><del><strong>Fixed Item 89</strong> Form validation - Error label that are appearing when validating checkboxes overlay the legend text. <a href="https://wet-boew.github.io/themes-dist/GCWeb/demos/formvalid/formvalid-en.html">Form validation</a>. How to reproduce: 1. Go in the form validation example page. 2. Click on submit to trigger the form validation plugin. 3. See the error label are over the legend text. - There probably a fix for it in the PR #8397</del></li>
```

Comment:
This URL will need to updated for "gcweb-compiled-demo" repository instead of themes-dist.

Reply:
Did that repo's WET demos go missing?

I can see some méli-mélo demos here:
* https://wet-boew.github.io/gcweb-compiled-demos/m%C3%A9li-m%C3%A9lo-demos/gc-th%C3%A9matique/th-canadaday/canada-day.html

But all the WET demos seem to be missing:
* https://wet-boew.github.io/gcweb-compiled-demos/index.html#wet-boew (GCWeb's home page links towards this)
* https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/footnotes/footnotes-en.html (my guess of where the footnotes plugin's demos is supposed to be)